superset-frontend/packages/superset-ui-core/src/validator/validateMapboxStylesUrl.ts

```diff
@@ -21,16 +21,22 @@ import { t } from '../translation';
 
 /**
  * Validate a [Mapbox styles URL](https://docs.mapbox.com/help/glossary/style-url/)
+ * or a title server URL.
  * @param v
  */
 export default function validateMapboxStylesUrl(v: unknown) {
   if (
     typeof v === 'string' &&
     v.trim().length > 0 &&
-    v.trim().startsWith('mapbox://styles/')
+    (v.trim().startsWith('mapbox://styles/') ||
+      v.trim().startsWith('tile://http') ||
+      v.trim().includes('osm') ||
```

Comment:
this seems a bit dangerous, seems startswith `https://tile.osm` or `https://tile.openstreetmap` might be safer. There's probably a fancy one-liner-type expression like `allowedPrefix.some(s => v.startswith(s)))`